src/loader.js

```diff
@@ -115,6 +118,39 @@ function setupModuleLoader(window) {
           _runBlocks: runBlocks,
 
           /**
+           * @ngdoc method
+           * @name angular.Module#info
+           * @module ng
+           *
+           * @param {Object=} info Information about the module
+           * @returns {Object|Module} The current info object for this module if called as a getter,
+           *                          or `this` if called as a setter.
+           *
+           * @description
+           * Additional info about this module
```

Comment:
This makes it sound as if passing an object would extend the existing info (not overwrite it).
I think we should be pretty explicit about that.